### PR TITLE
Akka Projection 1.0.0-RC3 announcement

### PR DESCRIPTION
**Pull request by renatocaval**

### CODE REVIEW

File: blog/2020-09-01-akka-projection-1.0.0-RC3-released.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Akka Projections 1.0.0-RC3 Released
+author: Renato Cavalcanti
+short: Please meet Akka Projections 1.0.0-RC3
+category: news
+tags: [releases]
+---
+
+Dear Hakkers,
```
Comment (ignasi35) [inline blog/2020-09-01-akka-projection-1.0.0-RC3-released.md:10]: Apparently we use `Dear Hakkers` and `Dear hakkers` interchangeably. 
(nothing to do here, just saying)

https://akka.io/blog/news-archive.html

File: blog/2020-09-01-akka-projection-1.0.0-RC3-released.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+We're happy to release the latest release candidate of Akka Projections, version 1.0.0-RC3. If no serious bug is reported this RC3 will become 1.0.0 GA next week.
```
Comment (ignasi35) [inline blog/2020-09-01-akka-projection-1.0.0-RC3-released.md:11]:
```suggestion
We're happy to announce the latest release candidate of Akka Projections, version 1.0.0-RC3. If no serious bug is reported this RC3 will become 1.0.0 GA next week. 
```

**Review by ignasi35 — APPROVED** [review]
Minor change `s/release/announce`. LGTM otherwise.

File: blog/2020-09-01-akka-projection-1.0.0-RC3-released.md (hunk 1 above)
Comment (patriknw) [reply blog/2020-09-01-akka-projection-1.0.0-RC3-released.md:10]: It should be `Dear hakkers,`
There have been other variations, such as hAkkers, but I'm pretty sure hakkers was what Jonas preferred and what Roland changed to.

Comment (renatocaval) [reply blog/2020-09-01-akka-projection-1.0.0-RC3-released.md:10]: I will change it, but this is a copy-and-paste from another one.

Comment (renatocaval) [inline blog/2020-09-01-akka-projection-1.0.0-RC3-released.md:10]:
```suggestion
Dear hakkers,
```

Comment (renatocaval) [reply blog/2020-09-01-akka-projection-1.0.0-RC3-released.md]: I changed it now

Comment (renatocaval) [reply blog/2020-09-01-akka-projection-1.0.0-RC3-released.md]: Btw, I was the author of a previous post in which I wrote "Hakkers" and then copy-and-pasted here. 

Fixed there as well, plus the file was on the wrong place also. Moved now.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**